tests/Proto.Cluster.CodeGen.Tests/ProtoGrainGenerationTests.cs

```diff
@@ -35,6 +32,8 @@ public void CanFindImportedNamespaces()
                 _testOutputHelper.WriteLine(codeFile.Text);
             }
 
+            var expectedOutput = File.ReadAllText(expectedOutputFile).Trim();
+            res.Single().Text.Trim().Should().Be(expectedOutput);
```

Comment:
it might be a bit too strict to test for exact whitespace, indentation etc.
touching either the template or expected output will fail the tests.
XUnit is pretty good at pointing out where strings differ though, so maybe not a big issue

Reply:
The idea was to have a guarantee against regressions. It will require us to change the expected result any time we change the generator output, but that is kind of the point :)

It would be nice to not have it so dependent on exact white space etc, do you have a suggestion for a better approach? I did consider something like parsing it with Roslyn and just checking the AST, but that is more work for something that might not give a better outcome overall.

Reply:
I was thinking something like remove all WS and compare the output. if that passes, but not the raw strings, then we know its just a whitespace issue, which would make it easier to diagnose at least. but might be overkill